cmd/files.go

```diff
@@ -536,8 +536,8 @@ func FileIgnore(blockID string) error {
 // ------------------------------------
 // > files key
 
-func FileKeys(targetID string) error {
-	res, err := executeJsonCmd(http.MethodGet, "keys/"+targetID, params{}, nil)
+func FileKeys(dataID string) error {
+	res, err := executeJsonCmd(http.MethodGet, "keys/"+dataID, params{}, nil)
```

Comment:
This should be documented in the changelog if this is a value change and not just a variable name change